spec/avram/transaction_spec.cr

```diff
@@ -66,6 +66,11 @@ describe "Avram::SaveOperation" do
         operation.saved?.should be_true
       end
     end
+
+    it "releases connection after operation finishes and no transactions open", tags: Avram::SpecHelper::TRUNCATE do
```

Comment:
Have to truncate instead of wrap the specs in a transaction so that there will not be any open transactions left on the connection when the operation finishes.